### PR TITLE
SS-3439 :  support all react-icons in Callout

### PR DESCRIPTION
**Pull request by AnkitRastogi3**

### CODE REVIEW

File: components/Callout.tsx [hunk 1]
```diff
@@ -27,15 +29,16 @@ const Callout = ({ title, icon, children }) => (
...
-            return <LinkTwoIcon style={iconStyle} />;
-        default:
-            return <InfoIcon style={iconStyle} />;
-    }
+interface DynamicIconProps {
+    iconName: string;
+}
+
+const DynamicIcon: React.FC<DynamicIconProps> = ({ iconName }) => {
+    const Icons = dynamic(() => import(`@100mslive/react-icons/dist/${iconName}.js`) as any, {
```
Comment (KaustubhKumar05) [inline components/Callout.tsx:37]:
```suggestion
    const Icon = dynamic(() => import(`@100mslive/react-icons/dist/${iconName}.js`) as any, {
```

File: components/Callout.tsx [hunk 2]
```diff
@@ -17,7 +19,7 @@ const Callout = ({ title, icon, children }) => (
         }}>
         <Box css={{ m: '20px 24px' }}>
             <Flex align="center" css={{ color: '$textHighEmp', gap: '$2', mb: '$2' }}>
-                {getIcon(icon)}
+                <DynamicIcon iconName={icon} />
```
Comment (KaustubhKumar05) [inline components/Callout.tsx:22]: `name` is fine, `iconName` is redundant

Comment (KaustubhKumar05) [reply components/Callout.tsx:22]: The argument can be iconName instead of icon

File: components/Callout.tsx [hunk 3]
```diff
@@ -27,15 +29,16 @@ const Callout = ({ title, icon, children }) => (
...
-        default:
-            return <InfoIcon style={iconStyle} />;
-    }
+interface DynamicIconProps {
+    iconName: string;
+}
+
+const DynamicIcon: React.FC<DynamicIconProps> = ({ iconName }) => {
+    const Icons = dynamic(() => import(`@100mslive/react-icons/dist/${iconName}.js`) as any, {
+        loading: () => <InfoIcon style={iconStyle} />
```
Comment (KaustubhKumar05) [inline components/Callout.tsx:38]: Can we add this as a fallback? It's possible that some icon can get renamed/removed in the fututre

File: components/Callout.tsx [hunk 4]
```diff
@@ -27,15 +29,22 @@ const Callout = ({ title, icon, children }) => (
...
+const DynamicIcon: React.FC<DynamicIconProps> = ({ name }) => {
+    let Icon;
+    try {
+        Icon = dynamic(() => import(`@100mslive/react-icons/dist/${name}.js`) as any, {
+            loading: () => <InfoIcon style={iconStyle} />,
+            ssr: false // Disable server-side rendering for dynamic imports
+        });
+    } catch (error) {
+        console.error(`Error loading icon module for "${name}":`, error);
+        Icon = () => <InfoIcon style={iconStyle} />;
```
Comment (KaustubhKumar05) [inline components/Callout.tsx:45]: Has this been tested?

Comment (AnkitRastogi3) [reply components/Callout.tsx:45]: Yes

**Pull request merged**